pkg/hostagent/hostagent.go

```diff
@@ -237,17 +237,13 @@ func writeSSHConfigFile(sshPath, instName, instDir, instSSHAddress string, sshLo
 	return os.WriteFile(fileName, b.Bytes(), 0o600)
 }
 
-func determineSSHLocalPort(confLocalPort int, instName string) (int, error) {
+func determineSSHLocalPort(confLocalPort int) (int, error) {
 	if confLocalPort > 0 {
 		return confLocalPort, nil
 	}
 	if confLocalPort < 0 {
 		return 0, fmt.Errorf("invalid ssh local port %d", confLocalPort)
 	}
-	if instName == "default" {
```

Comment:
Did we want to keep this functionality, for instances created before 2.0.0? To not break any existing ones?

https://github.com/lima-vm/lima/issues/3712#issuecomment-3133506245